frontend-gd/src/main/pages/new-date-page/new-date-page.js

```diff
@@ -0,0 +1,124 @@
+// React Imports
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+
+// Globals
+import variables from '../../tools/variables';
+import Controller from '../../tools/Controller';
+
+// Modules && General Components
+import LoggedInNav from '../../modules/nav-bar/logged-in-navbar';
+import NavbarBootstrap from '../../modules//nav-bar/navbar-bootstrap';
+
+// Component Specfic Imports
+import NewDateForm from './components/new-date-form'
+
+// Styles
+import '../../../App.scss';
+
+// Documentation/Notes
+
+const API = process.env.NODE_ENV === 'production' ? 'https://the-great-date-app.herokuapp.com' : 'http://localhost:3000'
```

Comment:
Is there a place we can put this const API and import it in? We have it on almost every page.

Reply:
Yeah it's in the request.js file, we don't need it anywhere else. So long as we use the Controller then we're good.